src/components/cylc/workflow/Toolbar.vue

```diff
@@ -38,88 +38,87 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     </v-btn>
     <!-- title -->
     <v-toolbar-title
-      class="text-md-h6 text-subtitle-1"
+      class="c-toolbar-title text-md-h6 text-subtitle-1 font-weight-medium text-primary"
+      :class="showNavBtn ? 'ml-0' : null"
     >
-      <span class="c-toolbar-title">{{ title }}</span>
+      {{ title }}
     </v-toolbar-title>
 
     <!-- control bar elements displayed only when there is a current workflow in the store -->
     <template v-if="currentWorkflow">
       <div class="c-workflow-controls flex-shrink-0">
-        <v-icon
+        <v-btn
           id="workflow-mutate-button"
-          color="#5E5E5E"
           v-cylc-object="currentWorkflow"
-        >
-          {{ svgPaths.menu }}
-        </v-icon>
+          :icon="svgPaths.menu"
+          size="small"
+        />
 
-        <v-icon
+        <v-btn
           id="workflow-play-button"
-          color="#5E5E5E"
```

Comment:
Why? I think the black colour helps distinguish from disabled state

Reply:
The solid black is too harsh which is why it was toned down.

Reply:
A bunch of icons are now defaulting to solid black which is jarring with the overall design.

*Before:*

![Screenshot from 2023-05-11 15-19-15](https://github.com/cylc/cylc-ui/assets/16705946/ac3bc238-89ab-4752-a52c-f1960b0e90ee)

*After:*

![Screenshot from 2023-05-11 15-19-04](https://github.com/cylc/cylc-ui/assets/16705946/f3e09275-b8eb-4752-add8-bd78e4cbdc23)

Reply:
I've looked into it and Vuetify 2 had a default color for all `v-icon`s whereas Vuetify 3 leaves it up to DOM inheritance (i.e. font color). Ironically, the added `#5E5E5E` made it _darker_ than the default color! Now that I'm used to the dark icons it doesn't seem jarring at all, if we want to have a comb through I think it would best to in a follow-up PR?

Reply:
For the default, it looks like multiple Vuetify components override the `body { color }` e.g. the draw and toolbars so this is just the pain of trying to bend Vuetify to match our design sketches, fair enough.

Reply:
The Vuetify update is a style change anyway (all the icons have changed colour), so I don't see this as sneaking something through. Rather than leaving behind an inconsistent icon colour, I think the better option is to open an issue for deciding what colour the icons should be?

Reply:
I disagree, but I'll cave in.